apps/challenges/views.py

```diff
@@ -927,15 +927,18 @@ def create_challenge_using_zip_file(request, challenge_host_team_pk):
             for email in emails:
                 user = User.objects.get(email=email)
                 host = Participant(
-                            user=user,
-                            status=Participant.ACCEPTED,
-                            team=participant_host_team,
-                    )
+                    user=user,
```

Comment:
@yashdusing 
I think the editor or Automatic PEP8 Formatter such as `autopep8` has made a lot of unnecessary changes which are not needed in this PR.

Reply:
Yeah, I’ll clean that up in a while 😅